draft-ietf-core-resource-directory.md

```diff
@@ -232,47 +231,44 @@ The Resource Directory is primarily a tool to make discovery operations more
 efficient than querying /.well-known/core on all connected devices, or across
 boundaries that would be limiting those operations.
 
-It provides a cache (in the high-level sense, not as defined in
-{{RFC7252}}/{{?RFC2616}}) of data that could otherwise only be obtained by
-directly querying the /.well-known/core resource on the target device, or by
-accessing those resources with a multicast request.
+It provides information about resources hosted by other devices that could otherwise only be obtained by
+directly querying the /.well-known/core resource on these other devicess, either by a unicast request or a multicast request.
 
 Only information SHOULD be stored in the resource
-directory that is discoverable from querying the described device's
+directory that is obtained by querying the described device's
```

Comment:
I liked the older wording better here: The new one prescribes a course of action (that's only actually taken in simple registration).
```suggestion
directory that can be obtained by querying the described device's
```